### ISSUE TITLE
How do I make this white line disappear when using this library ?

### ISSUE THREAD

**Issue by LeonardBonfils** [gontovnik/DGElasticPullToRefresh#11]
Any idea how I can get the white line between the refresh view and my "Volume XX - Issue XX" label ? 

https://youtu.be/kM7qgGSWmWw

**Comment by gontovnik** [comment]
Set background color of the view to black? Or move your "Volume / Issue" view 1 pixel up?

**Comment by LeonardBonfils** [comment]
Ok so I couldn’t really move my label 1 pixel up. Didn’t really know how to do that since I made the label with Storyboards and the view was kind of “stuck”, as in it was a pre-set position.

I tried to set different views to black to see which one worked (in Storyboards).

In the end, I just did it programatically with:

self.view.backgroundColor = UIColor.blackColor() in my viewWillAppear(). The annoying thing is that it also makes my tableView black. Any idea on how to select just the “background” view, and not the entire view ?

From: Danil Gontovnik
Reply-To: gontovnik/DGElasticPullToRefresh
Date: Wednesday, November 25, 2015 at 5:36 PM
To: gontovnik/DGElasticPullToRefresh
Cc: Léonard Bonfils
Subject: Re: [DGElasticPullToRefresh] How do I make this white line disappear when using this library ? (#11)


Set background color of the view to black? Or move your "Volume / Issue" view 1 pixel up?
